scripts/irods/test/test_ifsck.py

```diff
@@ -83,3 +83,31 @@ def test_ifsck_continues_when_user_cannot_access_subdirectory__issue_5358(self):
             for d in dirs:
                 os.chmod(d, 0o777)
 
+    @unittest.skipIf(test.settings.RUN_IN_TOPOLOGY, "Skip for Topology Testing, checks vault")
+    def test_ifsck_unregistered_file_detected__6467(self):
+        directory_name = 'test_ifsck_unregistered_file_detected__6467'
+        filename1 = 'test_ifsck_unregistered_file_detected__6467_f1'
+        filename2 = 'test_ifsck_unregistered_file_detected__6467_f2'
+        full_directory_path = '%s/%s' % (os.getcwd(), directory_name)
+        full_directory_logical_path = '%s/%s' % (self.admin.session_collection, directory_name)
```

Comment:
Prefer testing with a normal user instead of an admin.

Running as a normal user helps uncover more issues.

_This applies to the other places where self.admin is used too._

Reply:
The problem is that I think these particular tests need to be run as an admin user.  The recursive register fails as another user even if I give that user permission to read the files.